### PR TITLE
Issue #102- Nested left navigation for tasks menu

### PR DESCRIPTION
**Pull request by cloudbehl**
- Added Admin menu and task as its sub-menu
- Active functionality in the sub-menus

### CODE REVIEW

**Comment by cloudbehl** [comment]
@gnehapk @kamleshverma1 Please review.

File: src/modules/base/navigation/menu-svc.js [hunk 1]
```diff
@@ -10,41 +10,57 @@
...
+            subMenus: [{
+                label: 'Tasks',
+                id: 'task',
+                href: 'task',
+                icon: 'fa fa-cog',
+                active: false
+            }]
         }];
 
         vm.setActive = function(menuId) {
```
Comment (gnehapk) [inline src/modules/base/navigation/menu-svc.js]: @cloudbehl Where are we calling setActive funtion to make the submenu active?

Comment (cloudbehl) [inline src/modules/base/navigation/menu-svc.js]: @gnehapk it is getting called from index.js.

**Comment by kamleshverma1** [comment]
@cloudbehl Have you tested this properly ?

**Comment by kamleshverma1** [comment]
@cloudbehl I can see there is conflict . please fixed the conflict

**Comment by cloudbehl** [comment]
@kamleshverma1 yes I have tested it, Also regarding the conflict can you please check again.

**Pull request merged**